bot/start.ts

```diff
@@ -195,7 +195,7 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
         throw new Error(ctxUpdateAssertMsg);
       }
       const tgUser = ctx.update.message.from;
-      if (!tgUser.username) return await messages.nonHandleErrorMessage(ctx);
+      if (tgUser.username === undefined) return await messages.nonHandleErrorMessage(ctx);
```

Comment:
@Mersho please don't change this line in the refactoring commit to eliminate ! usage when not boolean; I told you that this refactoring should ONLY affect lines that have been changed by the JS->TS conversion